tools/ReplayAnalysis/README.md

```diff
@@ -20,6 +20,14 @@ fields @message
 | filter @message like /can be used in Replay Analysis./
 ```
 * Pre-install Node js (Minimum Node >= 14 required for React)
+  * If you do not have Node js already installed, follow steps below:
+    * If using EC2 instance, run following commands in order:
+      * `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash`
+      * `source ~/.bashrc`
+      * `nvm install --lts`
+      * Check that Node js is installed by running: `node -e "console.log('Running Node.js ' + process.version)"`
```

Comment:
Can we not simply run a bash command like ```which node``` or something ?

Reply:
makes sense, just made a change